torchci/pages/flakytest.tsx

```diff
@@ -59,7 +70,9 @@ export default function Page() {
           handleSubmit={(s) => setURL(name, suite, s, limit)}
         />
       </h3>
-      {data === undefined ? (
+      {!hasSearch ? (
+        <div>Please search for something more specific</div>
```

Comment:
Feels a bit rude to have this be the first thing people see when they go to `/flakytest`

How about updating changing this to instructions on how to edit of the parameters up top?  Some folks still don't realize that you can click those funny looking text fields

Side note: We should update our UI so that text boxes actually look like text boxes

Reply:
I have updated the instructions to be more helpful